src/App.jsx

```diff
@@ -14,70 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const goodsObj = goods
+  .map((item, index) => ({ nameGoods: item, id: index + 1 }));
```

Comment:
Id should be some unique value, not just id + 1. You can use some tool like uuid to generate a unique id or just use the string as id as all the goods are unique in this task
https://www.npmjs.com/package/uuid